marklogic-data-hub-central/ui/e2e/cypress/fixtures/users/developer.json

```diff
@@ -2,5 +2,5 @@
     "user-name": "dh-dev",
     "description": "A data hub dev user",
     "password": "dh-dev",
-    "role": ["data-hub-developer"]
+    "role": ["data-hub-developer", "hub-central-entity-exporter"]
```

Comment:
Do the Cypress tests actually hit ML? I thought UI tests were just operating against mocks. 

If they do hit ML, I recommend having a generic "test user" and then enhancing the test framework so that it's easy to change roles on that user. That's something we have in the HC JUnit tests.

Reply:
Yes, Cypress hits ML. Since we deploy data-hub project with all the test users and data before running tests it may be difficult to achieve that at test running stage. I will sync with Srikanth about that.